OpenSim/Sandbox/MatlabScripts/createActuatorsFile.m

```diff
@@ -0,0 +1,156 @@
+function createActuatorsFile
+%createActuatorsFile   make and Print an OpenSim Actuator File from a Model
+%
+%  createActuatorsFile attempts to make a template actuators file that can
```

Comment:
"attempts" on L4 and "tries" on L5 imply that the function may fail. In what situations will failure occur? What are the prerequisites for success?